lib/index.d.ts

```diff
@@ -137,6 +137,16 @@ declare namespace Realm {
      * @see { @link https://realm.io/docs/javascript/latest/api/Realm.Object.html }
      */
     interface Object {
+        /**
+         * @returns returns an array of a given Realm Object property names
```

Comment:
```suggestion
         * @returns An array of the names of the object's properties.
```

Reply:
I tried to mimic this one. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys
Also from the TypeScript definitions 
/**
     * Returns the names of the enumerable string properties and methods of an object.
     * @param o Object that contains the properties and methods. This can be an object that you created or an existing Document Object Model (DOM) object.
     */
    keys(o: object): string[];

I think these are an example of good explanations but I can change it to your suggestion if you still think it's better

Reply:
My main objection is the double "@returns returns" and since its an instance method on a `Realm.Object` I think it is better to write "the object" instead of "a given Realm Object", because - it's not given by the user as with `Object.keys()`

Reply:
good point.